spec/helpers/walks_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
Remove the pending tests from the spec files you generate. They'll mess up things with Travis CI.